etc/bin/run-validate-tutorials

```diff
@@ -19,11 +19,13 @@
 set -eu
 cd "$(dirname "$0")"
 
-for FILE in $(echo ../../cylc/flow/etc/tutorial/*/.validate) ; do
-    echo "running $FILE"
-    (
-        cd "$(dirname "$FILE")"
-        ./.validate
-    )
+for DIR in  examples; do
```

Comment:
Shellcheck flagging this for loop only iterates once

Reply:
Ach, temp diff not supposed to have been committed.

```suggestion
for DIR in tutorials examples; do
```